src/content/docs/en/guides/testing.mdx

```diff
@@ -43,6 +44,30 @@ export default getViteConfig(
 
 See the [Astro + Vitest starter template](https://github.com/withastro/astro/tree/latest/examples/with-vitest) on GitHub.
 
+## Vitest and Container API
+
+<Since v="4.9.0" />
+
+You can natively test Astro components using the new [container API](/en/reference/container-reference/). First, setup [`vitest` as explained above](#vitest), then create a `.test.js` file to test your component:
```

Comment:
```suggestion
You can natively test Astro components using the [container API](/en/reference/container-reference/). First, setup [`vitest` as explained above](#vitest), then create a `.test.js` file to test your component:
```
If we don't say "new", then we don't have to remember to update whenever we decide it's not "new" anymore! :smile: